compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1930,6 +1930,44 @@ impl Arch {
     }
 }
 
+crate::target_spec_enum! {
+    pub enum Vendor {
+        Amd = "amd",
+        Apple = "apple",
+        Espressif = "espressif",
+        Fortanix = "fortanix",
+        Ibm = "ibm",
+        Kmc = "kmc",
+        Mti = "mti",
+        Nintendo = "nintendo",
+        Nvidia = "nvidia",
+        Openwrt = "openwrt",
+        Pc = "pc",
+        Risc0 = "risc0",
+        Sony = "sony",
+        Sun = "sun",
+        Unikraft = "unikraft",
+        Uwp = "uwp",
+        Vex = "vex",
+        Win7 = "win7",
+        Wrs = "wrs",
+    }
+    /// The vast majority of the time, the compiler deals with a fixed set of
```

Comment:
This comment is repeated multiple times with only slight variations. I think it would be fine to instead say "// See `enum Target` for why this `Unknown` variant exists."  (The `OS` type has an extra sentence that can be kept.)

Reply:
Moved, thanks.